packages/client/Atmosphere.ts

```diff
@@ -286,7 +286,17 @@ export default class Atmosphere extends Environment {
 
   handleFetch: FetchFunction = (request, variables, _, uploadables) => {
     return Observable.create((sink) => {
-      this.handleFetchPromise(request, variables, uploadables, sink)
+      const verboseSink = Object.assign({}, sink, {
+        error: (e, isUncaughThrownError) => {
+          if (e.message === 'No payload received') {
+            console.info('request:', request)
+            console.info('variables:', variables)
+            console.info('auth token:', this.authToken)
```

Comment:
@tiffanyhan are these going to be logged in Sentry? To make sure we catch that we can also use: https://docs.sentry.io/platforms/javascript/enriching-events/breadcrumbs/ 
Also, I think it'd be better to avoid console logging auth token. Sentry will tell us who's affected by this issue anyway.  Let me know what do you think!

Reply:
yeah i assume they will be logged bc `Error: No payload received` is logged as a breadcrumb in sentry when it's logged in the lib code.

i can definitely use the explicit API which will keep our server logs cleaner. The only problem is, now i can't test locally. i set the `SENTRY_DSN` to dev value locally, but when i check [sentry](https://sentry.io/organizations/parabol/issues/?project=107915&query=&statsPeriod=14d) i can see server errors being sent to sentry but not client errors. and i can see the correct dsn env value set on the window object... 🤔

i had the same thought with the authToken since we already see userId in sentry, but was just going by the AC. happy to remove the authToken part.

Reply:
hmm! the AC part of logging auth token is a good question, @mattkrick do we really want to do it? 
Regarding local testing, I think it _should_ work. One way or another, this error probably will be hard to reproduce locally 😞